README.md

```diff
@@ -159,6 +159,14 @@ Identity type, for example, could provide `Id` as its type representative:
 If a type provides a type representative, each member of the type must have
 a `constructor` property which is a reference to the type representative.
 
+
+## `forAll`
```

Comment:
Might “Parametricity” be a better heading?

Reply:
I don't think so.  If someone doesn't understand this signature because of the `forall` clause:

```js
map :: forall a b. Functor f => f a ~> (a -> b) -> f b
```

and tries to figure out what it is, a header that matches the confusing keyword is better than a term that -- given this confusion -- is likely unknown to them also.

Reply:
Good point. Shall we use `forall` rather than `forAll`, though?

Reply:
Absolutely.  I'll fix that tonight.